src/librustc/mir/interpret/allocation.rs

```diff
@@ -32,8 +32,10 @@ pub enum CheckInAllocMsg {
 }
 
 impl Display for CheckInAllocMsg {
+    /// When this printed as an error the context looks like this
+    /// "{test name} test failed: pointer must be in-bounds at offset..."
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{}", match *self {
+        write!(f, "{} test", match *self {
```

Comment:
This still prints "Memory access test", which is weird. Can you move "test" down into the `match` to where it makes sense?